__init__.py

```diff
@@ -29,7 +29,9 @@
 if platform.system() == "Windows":
     from .windows.windows import GrabImage, CopyImage
 elif platform.system() == "Linux":
-    from .linux.linux import GrabImage, CopyImage
+    from .linux import GrabImage, CopyImage
```

Comment:
Wait, is it good idea to remove the module name? I think it should be:
```suggestion
    from .linux.linux import GrabImage, CopyImage
```